packages/api/src/command/medical/facility/get-facility.ts

```diff
@@ -1,12 +1,10 @@
+import { Facility } from "../../../domain/medical/facility";
 import NotFoundError from "../../../errors/not-found";
 import { FacilityModel } from "../../../models/medical/facility";
 
 type GetFacilitiesQuery = Pick<FacilityModel, "cxId"> & Partial<{ ids: FacilityModel["id"][] }>;
 
-export const getFacilities = async ({
-  cxId,
-  ids,
-}: GetFacilitiesQuery): Promise<FacilityModel[]> => {
+export const getFacilities = async ({ cxId, ids }: GetFacilitiesQuery): Promise<Facility[]> => {
```

Comment:
Straightforward update, just moving to a more abstract type.

This is one of those: "if the code compiles it works".

Here, Organization and Patient.